handbook/support/p4-enablement.md

```diff
@@ -0,0 +1,163 @@
+# Dogfood Perforce server
+
+Perforce is a version control system like Git, subversion, or mercurial. While git is based on a distributed, decentralised model, Perforce is centralised. For testing purposes, you may use our Perforce dogfood server.
+
+## Setting up
+- To connect to the Perforce server, you'll need the Perforce cli installed locally. Use the command: `brew install --cask perforce`
+- The following environment variables configure your shell to point at the Perforce server. Set them to your `env` with the `export` command, or add them to your `.bashrc` or `.zshrc` file.
+```
+# .zshrc or .bashrc
+
+P4PORT=perforce.sgdev.org:1666 # This points the p4 cli at the dogfood server
```

Comment:
Could we also add a link here to the actual VM that is running Perforce?

Reply:
Happy to add info about this

@unknwon Is the perforce server hosted on GCP? If so, can you let us know the project name?

Reply:
See here for info about the deployment https://k8s.sgdev.org/github.com/sourcegraph/infrastructure/-/tree/dogfood/kubernetes/tooling/perforce

Reply:
Thanks Dave, adding links